11.PureComponent.md

```diff
@@ -0,0 +1,11 @@
+**Jam3 Lessons - React**
+
+# React.PureComponent
+
+> React.PureComponent is exactly like `React.Component` but implements `shouldComponentUpdate()` with a shallow prop and state comparison.
+
+> If your React component's `render()` function renders the same result given the same props and state, you can use `React.PureComponent` for a performance boost in some cases.
+
```

Comment:
I used quotes because I copied it from the React docs, and I think that's the best term to describe this. @jeremenichelli